index.rst

```diff
@@ -1,83 +1,120 @@
-==========
-User Guide
-==========
+========================
+{Singularity} User Guide
+========================
 
 
-Welcome to the Singularity User Guide!
+Welcome to the {Singularity} User Guide!
 
-This guide aims to give an introduction to Singularity, brief
+This guide aims to give an introduction to {Singularity}, brief
 installation instructions, and cover topics relevant to users building
 and running containers.
 
 For a detailed guide to installation and configuration, please see the
-separate Admin Guide for this version of Singularity at
+separate Admin Guide for this version of {Singularity} at
 `<\{admindocs\}/>`__.
 
-              
+
 Getting Started & Background Information
 ========================================
-              
+
 .. toctree::
    :maxdepth: 2
               
    Introduction to Singularity <introduction>
    Quick Start <quick_start>
    Security in Singularity <security>
 
-Building Containers   
+Building Containers
 ===================
 
+Learn how to write a definition file that can be used to build a
+container. Understand the environment within a build, how to perform
+remote builds, and how to use the ``--fakeroot`` feature to build as a
+non-root user.
+
 .. toctree::
-   :maxdepth: 3
-              
+   :maxdepth: 1
+
    Build a container <build_a_container>
    The Definition File <definition_files>
    Build Environment <build_env>
-   Singularity and Docker <singularity_and_docker>
    Fakeroot feature <fakeroot>
 
-Signing & Encryption
-====================
+Container Signing & Encryption
+==============================
+
+{Singularity} allows containers to be signed using a PGP key. The
+signature travels with the container image, allowing you to verify
+that the image is unmodified at any time. Encryption of containers
+using LUKS2 is also supported. Encrypted containers can be run without
+decrypting them to disk first.
 
 .. toctree::
-   :maxdepth: 2
+   :maxdepth: 1
 
    Sign and Verify <signNverify>
    Key management commands <key_commands>
    Encrypted Containers <encryption>
-   
+
 Sharing & Online Services
 =========================
 
+HPCng offers a suite of container services, with a free tier and
```

Comment:
No, it is Sylabs that offers those services.  Just leave this part of the changes out.

Reply:
I removed paragraph